deny.toml

```diff
@@ -201,29 +201,6 @@ name = "cbindgen"
 version = "*"
 reason = "cbindgen is always a dev dependency for us and we don't care if it pulls in other versions of things"
 
-[[bans.skip]]
-name = "thiserror"
-version = "1.0.69"
-reason = "temporarily allowing this. Remove when default-net is updated"
```

Comment:
Commit description is wrong, we still have this conflict:

```
warning[duplicate]: found 2 duplicate entries for crate 'thiserror'
    ┌─ /home/runner/_work/dataplane/dataplane/Cargo.lock:437:1
    │  
437 │ ╭ thiserror 1.0.69 registry+https://github.com/rust-lang/crates.io-index
438 │ │ thiserror 2.0.17 registry+https://github.com/rust-lang/crates.io-index
```

The reason we can remove these exceptions is that we've stopped blocking on duplicate crates a while ago, it was too much work to work around it. I'm good to remove these skips, but the commit description should be fixed.